MediaGalleryUi/view/adminhtml/web/js/action/deleteImage.js

```diff
@@ -58,6 +58,17 @@ define([
             });
         },
 
+        /**
+         * Returns content
+         *
+         * @param {string} recordPath
+         * @param {string} recordRelatedContentCount
+         * @return string
+         */
+        getContentMessage: function(recordPath, recordRelatedContentCount) {
+            return $.mage.__('This image is used in '+ recordRelatedContentCount +' page(s). Are you sure you want to delete "' + recordPath + '" image?');
```

Comment:
Image also can be used in blocks and products as well, i think we should create a proper message for all cases like in 1 page-pages block-blocks product-products.

Reply:
@Nazar65  could i use message like: "This image is used in 2 cms_page(s)  or (cms_block, catalog_category)" ?

Reply:
I think it can be the same behaviour, as like this displayed in view details page

Reply:
@Nazar65  created message for all cases. For example: 

![Screenshot 2020-06-01 at 09 52 20](https://user-images.githubusercontent.com/29776509/83402711-eadc6b00-a40f-11ea-825b-9b686f5d8909.png)
![Screenshot 2020-06-01 at 09 52 40](https://user-images.githubusercontent.com/29776509/83402714-ec0d9800-a40f-11ea-9e88-6382581ae7eb.png)